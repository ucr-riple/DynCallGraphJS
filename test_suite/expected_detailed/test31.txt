{
    'global (test31.js:1:1:11:3)': [
        'anon (test31.js:1:2:4:2)'
    ],
    'anon (test31.js:3:3:3:14)': [
        'anon (test31.js:5:3:7:4)'
    ]
}