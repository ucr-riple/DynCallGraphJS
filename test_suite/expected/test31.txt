{ '(test31.js:1:1:11:3)':
   [ '(test31.js:1:2:4:2)' ],
  '(test31.js:3:3:3:14)':
   [ '(test31.js:5:3:7:4)' ] }